snoop/trace.py

```diff
@@ -7,11 +7,11 @@
 from opencensus.stats import measure as measure_module
 from opencensus.stats import stats as stats_module
 from opencensus.stats import view as view_module
-from opencensus.stats.exporters import prometheus_exporter as prometheus
+from opencensus.ext.prometheus import stats_exporter as prometheus
```

Comment:
Can we remove the prometheus thing? AFAIK we're not using it, and it's causing trouble on nomad. @gabriel-v, @mugurrus, thoughts?

Reply:
I think the update is good, since it's just following the changelog. I vote to fix the whole tracing situation in another pull request and leave this patch as it is.